apps/explorer/src/types/assets.ts

```diff
@@ -16,39 +16,62 @@ You should have received a copy of the GNU Lesser General Public License
 along with the library. If not, see <http://www.gnu.org/licenses/>.
 */
 
-import { TokenInfo } from '@alephium/token-list'
-import { FungibleTokenMetaData, NFTMetaData } from '@alephium/web3'
+import { NFTTokenUriMetaData } from '@alephium/shared'
+import { FungibleTokenMetadata, NFTMetadata as NFTMetadataBase } from '@alephium/web3/dist/src/api/api-explorer'
 
 import client from '@/api/client'
 
 export type AssetType = Awaited<ReturnType<typeof client.node.guessStdTokenType>>
 
 export type AssetBase = { id: string; type: AssetType }
 
-export type FungibleTokenMetadata = Omit<FungibleTokenMetaData, 'totalSupply'> & { id: string; verified: boolean }
+export type FungibleTokenMetadataBase = Omit<FungibleTokenMetadata, 'totalSupply'> & {
+  verified: boolean
+  logoURI?: string
+}
+
+export type VerifiedFungibleTokenMetadata = TokenListTokenInfo & {
+  type: 'fungible'
+  verified: true
+}
+
+export type UnverifiedFungibleTokenMetadata = FungibleTokenMetadata & { type: 'fungible'; verified: false }
 
-export type VerifiedFungibleTokenMetadata = TokenInfo & { type: 'fungible'; verified: true }
+export type NFTMetadata = NFTMetadataBase & { id: string; type: 'non-fungible'; verified: boolean }
 
-export type UnverifiedFungibleTokenMetadata = TokenInfo & { type: 'fungible'; verified: false }
+export type NFTMetadataWithFile = NFTMetadata & { file?: NFTFile }
 
-export type UnverifiedNFTMetadata = NFTMetaData & { id: string; type: 'non-fungible'; verified: false }
+export type UnverifiedNFTMetadata = Omit<NFTMetadata, 'verified'> & { verified: false }
 
-export type UnverifiedNFTMetadataWithFile = UnverifiedNFTMetadata & { file: NFTFile }
+export type UnverifiedNFTMetadataWithFile = Omit<NFTMetadata, 'verified'> & { file?: NFTFile; verified: false }
 
 export type NumericTokenBalance = { balance: bigint; lockedBalance: bigint }
 
-export type NFTFile = {
+export type NFTFile = NFTTokenUriMetaData & {
   assetId: string
-  name?: string
+}
+
+// TODO: This has been extracted from the token-list. Cleanup types!
```

Comment:
Was there no way to import it from there?

Reply:
What is blocking us from simply importing it from `@alephium/token-list`? They seem identical: https://github.com/alephium/token-list/blob/master/lib/types.ts